mont_blanc/fastdds/CMakeSettings.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Is this a local file for your editor? It probably shouldn't be checked in.

Reply:
Fixed! https://github.com/osrf/zenoh_evaluation/pull/4/commits/46b942f22dbb135780ae7e9242cf7727f1a862b4